flocker/apiclient/_client.py

```diff
@@ -25,14 +25,14 @@ class Dataset(PClass):
     A dataset in the configuration.
 
     :attr UUID primary: The node where the dataset should manifest.
-    :attr int maximum_size: Size of new dataset, in bytes.
+    :attr maximum_size: Size of new dataset in bytes (as ``int``) or ``None``.
```

Comment:
Remove the word "new". There doesn't seem to be any constraint that this structure can only be used for a new dataset. It is used in the `move` and `list` methods later on.